python2verilog/simulation/display.py

```diff
@@ -17,7 +17,7 @@ def parse_stdout(stdout: str) -> Generator[tuple[str, ...], None, None]:
 
 
 def strip_signals(
-    actual_raw: Iterable[tuple[str, ...]],
+    actual_raw: Iterable[tuple[str, ...]], include_invalid: bool
 ) -> Generator[Union[tuple[int, ...], int], None, None]:
     """
     Implementation-specific (based on testbench output)
```

Comment:
> valid and wait

valid and ready

Reply:
> Only yields a row if both valid and ready

Something along the lines of "Only yields a row if the caller is ready for an output and the module produces an output", but also includes `include_invalid`

Reply:
Also, why not pass a `CodegenConfig`? Then you can have more configs for your tb